src/overlays/actors/ovl_En_Ma1/z_en_ma1.c

```diff
@@ -19,45 +36,412 @@ const ActorInit En_Ma1_InitVars = {
     (ActorFunc)EnMa1_Update,
     (ActorFunc)EnMa1_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Ma1/func_80AA0690.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Ma1/func_80AA0778.s")
+ColliderCylinderInit D_80AA1640 = {
+    0x0A, 0x00,       0x00, 0x39, 0x20,   0x01,   0x00,       0x00,   0x00,   0x00,   0x00,
+    0x00, 0x00000000, 0x00, 0x00, 0x00,   0x00,   0x00000000, 0x00,   0x00,   0x00,   0x00,
+    0x00, 0x00,       0x01, 0x00, 0x0012, 0x002E, 0x0000,     0x0000, 0x0000, 0x0000,
+};
+
+Sub98Init5 D_80AA166C = {
+    0x00, 0x0000, 0x0000, 0x0000, 0xFF,
+};
+
+struct_D_80AA1678 D_80AA1678[] = {
+    { 0x06000820, 1.0f, 0x00, 0.0f },
+    { 0x06000820, 1.0f, 0x00, -10.0f },
+    { 0x06008D64, 1.0f, 0x00, 0.0f },
+    { 0x06008D64, 1.0f, 0x00, -10.0f },
+};
+
+Vec3f D_80AA16B8 = { 800.0f, 0.0f, 0.0f };
+
+u32 D_80AA16C4[] = {
+    0x06001F18,
+    0x06002B18,
+    0x06002F18,
+};
+
+u32 D_80AA16D0[] = {
+    0x06001B18,
+    0x06002318,
+    0x06002718,
+};
+
+s32 D_80AA16DC = 0;
```

Comment:
just padding?

Reply:
same deal as this, no? https://github.com/zeldaret/oot/pull/72#discussion_r410367086

Reply:
Well the difference is that one had more data after it, but it doesn't look like this one does?

Reply:
Ah, okay let me see..